docs/governance/grants-bounties/udt-grantee-handbook.md

```diff
@@ -10,80 +10,89 @@ sidebar_position: 2
-for one of our UDT grants. This guide showcases applying for a grant, 
-establishing clear milestones, and provides a description of how a grant 
-application is processed and reviewed by the Grant Committee as well as the 
-milestone review and approval process.
+for a grant from the Unlock DAO. This guide showcases applying for a grant, 
+establishing clear milestones, and provides a description of how the grant process
+proceeds throughout the life of a grant.
 
 ## Eligibility
 
 Any educational program, integration, plugin, application, or event that will 
 increase the visibility, credibility, or Gross Network Product (GNP) of Unlock 
-Protocol may be eligible for a grant.
+Protocol should consider applying for a grant from the Unlock DAO.
 
-## Submitting an Application
+## What are grants?
 
-The Unlock Protocol UDT token grant program is administered using a management 
-tool called [Wonderverse](https://www.wonderverse.xyz/). This is the place where 
-you create your application, submit your milestone work for review, and 
-collaborate with the Unlock team and Grants Committee for support and resources.
+Grants are one of the features of the Unlock ecosystem. As Unlock continues to progressively decentralize the decision-making and governance of the protocol, streamlining and further decentralizing the grants process supports that goal.
 
-If you don't already have a Wonderverse account, you will need to create one. 
-Be sure to connect your wallet address so that, upon completion and approval, 
-your grant can be distributed to you! If you’re working in a team, be sure to 
-make the application with a team account or representative with the wallet your 
-team wishes to use to receive payments. (Pro-tip: If you already use Discord, 
-you can connect Discord with Wonderverse and be sent notifications of updated 
-grant activity so you don’t miss anything.)
-## Getting Started
+Historically, the Unlock Labs team was actively involved in the grants through both managing the process as well as distributing grants via Unlock Labs. Going forward, Unlock Labs recommends the Unlock DAO take a more active role in grant activity. The reason for this is simple: much more community involvement in grants, powered by the Unlock DAO itself.
 
-Head over to the [“Grants” pod](https://app.wonderverse.xyz/pod/68474324611236166/boards) 
-of the Unlock Protocol Wonderverse workspace. This is the place where you’ll 
-find general resources about grants and where the grants committee coordinates on 
-applications that are newly submitted and in review.
+Currently in early 2023, the [Unlock DAO’s timelock contract](https://etherscan.io/address/0x17eedfb0a6e6e06e95b3a1f928dc4024240bc76b) holds approximately 92,000 UDT governance tokens. These governance tokens can be distributed to members of the community as grants, if members of the DAO create proposals that pass the governance process. 
```

Comment:
I am usually a bit worried about putting numbers because I think we will forget to update them but I leave this to your judgement. if you think it is important we can leave it for now!

Reply:
Otherwise, we can add a sentence like Currently in early 2023, the Unlock DAO’s timelock contract holds the largest stake of UDT governance tokens.